src/main/java/uk/gov/hmcts/reform/hmc/Application.java

```diff
@@ -3,14 +3,13 @@
 import org.springframework.boot.LazyInitializationExcludeFilter;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
-import org.springframework.boot.autoconfigure.jdbc.DataSourceAutoConfiguration;
 import org.springframework.cloud.openfeign.EnableFeignClients;
 import org.springframework.context.annotation.Bean;
 import uk.gov.hmcts.reform.hmc.config.MessageReceiverConfiguration;
 
 import java.time.Clock;
 
-@SpringBootApplication(exclude = {DataSourceAutoConfiguration.class })
```

Comment:
Good reason for this change? Just a query.

Reply:
To enable database auto-configuration in Spring Boot removed the exclude condition. HMAN-578 requires Outbound to insert the hearing journey into database.